polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -97,7 +100,8 @@ def get_sqlalchemy_types(cls) -> dict[Any, Callable[[], Any]]:
             postgresql.TSTZRANGE: lambda: (cls.__faker__.past_datetime(), datetime.now()),  # noqa: DTZ005
             postgresql.HSTORE: lambda: cls.__faker__.pydict(),
             # `types.JSON` is compatible for sqlachemy extend dialects. Such as `pg.JSON` and `JSONB`
-            types.JSON: lambda: cls.__faker__.pydict(),
+            # accept `str,int,float` as basic types in case `sqlalchemy/sql/sqltypes.json.json_serializer` raise error
+            types.JSON: lambda: cls.__faker__.pydict(value_types=[int, str, float]),
```

Comment:
@guacs seems value_types limitation is not effected. I don't find how to fix it.

Reply:
Could we move this into a separate PR?

Reply:
sure

Reply:
revert it. I will put it in another PR. thanks